airflow/gcp/hooks/gcs.py

```diff
@@ -418,23 +431,19 @@ def create_bucket(self,
         :type resource: dict
         :param storage_class: This defines how objects in the bucket are stored
             and determines the SLA and the cost of storage. Values include
-
```

Comment:
I would also ask all other unrelated changes to the documentation to be withdrawn.